src/hooks/useNotificationsStore.js

```
@@ -0,0 +1,131 @@
+    } catch (err) {
+      console.debug?.('useNotificationsStore: markAsRead failed', err);
+    }
+  }, [db, user?.uid]);
+
+  const markVisited = useCallback(async (communityId) => {
+    if (!db || !user?.uid || !communityId) return;
+    try {
+      const prefRef = doc(db, 'users', user.uid, 'preferences', 'community');
+      try {
+        await updateDoc(prefRef, { [`lastVisited.${communityId}`]: serverTimestamp(), updatedAt: serverTimestamp() });
+      } catch (innerErr) {
```

ESLint

[error] 'innerErr' is defined but never used.

(no-unused-vars)